Packs/ZeroFox/Integrations/ZeroFox/ZeroFox_test.py

```diff
@@ -1301,3 +1307,35 @@ def test_get_alert_attachments_command(requests_mock, mocker):
     assert alert_id == alert_id_called
     assert isinstance(results.outputs, list)
     assert results.outputs_prefix == "ZeroFox.AlertAttachments"
+
+
+def test_get_compromised_credentials_command(requests_mock, mocker):
```

Comment:
This is just a suggestion:
Maybe it would be good to add a UT so that the response is 404 and then the test is on the readable_output